scripts/update-integrations.mjs

```diff
@@ -15,6 +15,38 @@ import {
 } from './integrations.mjs';
 import { markdownToPlainText } from './markdown.mjs';
 import { fetchDetailsForPackage, fetchDownloadsForPackage, searchByKeyword } from './npm.mjs';
+import _yargs from 'yargs'
+import { hideBin } from 'yargs/helpers';
```

Comment:
I think `yargs` is probably overkill here — this script is run by CI tasks 99.9% of the time. I think instead of adding an extra `fix-homepageurl` option it should just be part of the `unsafe` command. There aren’t really scenarios where you _don’t_ want that right? That way it’s still just a single flag and we do without `yargs`.